lib/facter/self_service.rb

```diff
@@ -240,4 +240,18 @@
     # Is puppet_metrics_collector::system configured
     { S0040: PuppetSelfService.service_running_enabled('puppet_system_processes-metrics.timer') }
   end
+
+  chunk(:S0034) do
+    # PE has not been upgraded / updated in 1 year
```

Comment:
This needs to be restricted to the primary.

Reply:
hmm that used to be there... its like the rebasing went back a little too far